plaidml/edsl/ffi.cc

```diff
@@ -114,8 +112,6 @@ void plaidml_edsl_init(  //
   ffi_wrap_void(err, [&] {
     std::call_once(is_initialized, []() {
       IVLOG(1, "plaidml_edsl_init");
-      registerAllDialects();
-      registerAllPasses();
```

Comment:
This is the key bit, we can't expect all the dialects/passes will be known to the EDSL lib. Instead we make it a linker problem. This way 3rd parties can create their own version of `libplaidml.so` that contains the targets they wish to include.